tests/test_paths.py

```
@@ -0,0 +1,51 @@
+"""Tests for runtime path resolution."""
+
+from pathlib import Path
+from unittest.mock import patch
+
+from biblebot import auth, cli, paths
+
+
+def test_default_paths_use_legacy_config_home():
+    """Default path model should remain ~/.config/matrix-biblebot."""
+    with patch.dict("os.environ", {}, clear=True):
+        with patch("pathlib.Path.home", return_value=Path("/home/testuser")):
```

Ruff (0.15.10)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)